_includes/releases/v22.2/v22.2.0-alpha.1.md

```diff
@@ -0,0 +1,851 @@
+- Views are no longer allowed to reference types that are defined in different databases. Even though this was allowed at view-creation time previously, it would cause errors, since cross-database type references are not supported. [#82763][#82763]
+- [`CREATE TABLE AS`](../{{site.versions["dev"]}}/create-table-as.html) in explicit transactions would fail with an error if the size of the source table exceeded the Raft command size limit. [#82951][#82951]
+- Range lease transfers are no longer permitted to follower replicas that may require a [Raft snapshot](../{{site.versions["dev"]}}/architecture/replication-layer.html#snapshots). This ensures that lease transfers are never delayed behind snapshots, which could previously create range unavailability until the snapshot completed. Lease transfers are now only allowed when the outgoing leaseholder can guarantee that the incoming leaseholder does not need a snapshot. [#82758][#82758]
+- Fixed the issue when creating a unique, expression index on a [`REGIONAL BY TABLE`](../{{site.versions["dev"]}}/multiregion-overview.html). See #83076. [#83125][#83125]
+- Fixed a bug where in rare cases a stale read could be returned. The fix is to introduce a new in-memory field to a `LeaseStatus`, which is when it most recently acquired data to a different store. Any uncertain observed timestamps before this time are ignored. [#83345][#83345]
+- Previously, the `querySummary` metadata field in the [`crdb_internal.statement_statistics`](../{{site.versions["dev"]}}/crdb-internal.html) table was inconsistent with the query metadata field for executed prepared statements. These fields are now consistent for prepared statements. [#83673][#83673]
+- Fixed a bug where [`BACKUP`](../{{site.versions["dev"]}}/backup.html) may be missing data when the cluster was configured with very low values for `kv.bulk_sst.max_allowed_overage` and `kv.bulk_sst.target_size` cluster settings. [#83102][#83102]
+- Fixed an issue where some [exports](../{{site.versions["dev"]}}/export.html) would receive `"unexpected closure of consumer"` rather than the actual error the export encountered. [#77938][#77938]
+- Fixed a bug causing a graceful node shutdown to stall forever. [#83824][#83824]
+- The `PASSWORD` option of the [`CREATE`](../{{site.versions["dev"]}}/create-role.html)/[`ALTER ROLE`](../{{site.versions["dev"]}}/alter-role.html) commands now require the password to be surrounded with single quotes. This fixes confusion that could arise when a mixed-case string is used, since previously that would cause the password to be normalized to lowercase. [#83924][#83924]
+- This patch fixes the `row_to_json` SQL function so it does not error out with input having the `VOID` data type. [#83876][#83876]
```

Comment:
Fixed a bug causing the `row_to_json` SQL function to error out when used with input having the `VOID` data type.